### ISSUE TITLE
warning message on `colima start`

### ISSUE THREAD

**Issue by joachim-n** [abiosoft/colima#802]
### Description

I see this error go past quickly when I do `colima start`:

```
a/bin/qemu-system-aarch64\" is not properly signed with the \"com.apple.security.hypervisor\" entitlement" error="binary \"/Users/joachim/.colima/_wrapper/3a9197e1ca3cd2da076da2b473d7a7eb118e2cca/bin/qemu-system-aarch64\" seems signed but lacking the \"com.apple.security.hypervisor\" entitlement"
> You have to sign the QEMU binary with the "com.apple.security.hypervisor" entitlement
```

### Version

Colima Version:
Lima Version:
Qemu Version:

### Operating System

- [ ] macOS Intel <= 12 (Monterrey)
- [ ] macOS Intel >= 13 (Ventura)
- [ ] macOS M1 <= 12 (Monterrey)
- [X] macOS M1 >= 13 (Ventura)
- [ ] Linux

### Output of `colima status`

INFO[0000] colima is running using QEMU
INFO[0000] arch: aarch64
INFO[0000] runtime: docker
INFO[0000] mountType: sshfs
INFO[0000] socket: unix:///Users/joachim/.colima/default/docker.sock

### Reproduction Steps

1. Do `colima start`

My initial start command was:

> colima start --cpu 4 --memory 6 --disk 100 --vm-type=qemu --mount-type=sshfs --dns=1.1.1.1

**Comment by rfay** [comment]
If you search the issue queue here and in [lima](https://github.com/lima-vm/lima/issues) you'll see that there are at least a dozen open issues about this. The maintainers of lima and qemu are very much aware of this problem. It's not actually something colima has control of, except via 
* https://github.com/abiosoft/colima/issues/635

**Comment by AkihiroSuda** [comment]
Duplicate of:
- https://github.com/abiosoft/colima/issues/796

**Comment by abiosoft** [comment]
This is fixed in v0.6.6

**Comment by rfay** [comment]
I assume fixed in v0.6.0 as opposed to v0.6.6?

**Comment by abiosoft** [comment]
@rfay It was a typo, thanks :)
